crates/ide_completion/src/item.rs

```diff
@@ -292,16 +309,22 @@ impl CompletionItem {
         self.deprecated
     }
 
-    pub fn relevance(&self) -> Relevance {
+    pub fn relevance(&self) -> CompletionRelevance {
         self.relevance
     }
 
     pub fn trigger_call_info(&self) -> bool {
         self.trigger_call_info
     }
 
-    pub fn ref_match(&self) -> Option<Mutability> {
-        self.ref_match
+    pub fn ref_match(&self) -> Option<(Mutability, CompletionRelevance)> {
```

Comment:
This change partially reverts a change in #7944, but I think it is necessary for us to return a `CompletionRelevance` here, because we don't want consumers of this API to have to figure out for themselves how relevant this `ref_match` is.

Reply:
My thinking was that ref match just *implies* high relevance, so we don't need a separate bit of info here. But, again, no super strong opinion here.